src/memory_tasks_backend.js

```diff
@@ -0,0 +1,31 @@
+var express = require('express')
+const TasksBackend = require('./tasks_backend')
+
+module.exports = class MemoryTasksBackend extends TasksBackend {
+  constructor () {
+    super()
+    this.tasks = []
+  }
+
+  async init (config) {
+    const router = await super.init(config)
+    router.use(express.json())
+
+    router.put('/', async (req, res) => {
+      await this.setTasks(req.body)
+      res.json(this.tasks)
+    })
+
+    return router
+  }
+
+  async * getTasks () {
+    for (const task of this.tasks) {
+      yield task
+    }
+  }
+
+  async setTasks (tasks) {
+    this.tasks = tasks || []
+  }
+}
```

Comment:
This looks good. We should make sure to comment this well, because this is probably going to be used in documentation of an example of a non-trivial `TasksBackend`

Reply:
Done! I'll request another review to cover that change.